web/src/routes/operations/+page.svelte

```diff
@@ -1264,21 +1276,21 @@
 	function updateAircraftMarkerFromAircraft(aircraft: Aircraft): void {
 		if (!map) return;
 
-		// Try to use latestLatitude/latestLongitude from aircraft object
-		if (aircraft.latestLatitude != null && aircraft.latestLongitude != null) {
+		// Try to use latitude/longitude from aircraft object
+		if (aircraft.latitude != null && aircraft.longitude != null) {
 			console.log('[MARKER] Using latest position from aircraft:', {
 				id: aircraft.id,
-				lat: aircraft.latestLatitude,
-				lng: aircraft.latestLongitude
+				lat: aircraft.latitude,
+				lng: aircraft.longitude
 			});
 
 			// Create a pseudo-fix from latest position
 			const pseudoFix: Fix = {
```

Comment:
There's no need for this "pseudo fix". we're getting "currentFix" from the aircraft object. that *is* a fix.